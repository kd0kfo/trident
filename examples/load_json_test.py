#!/usr/bin/env python

import json
j = json.loads(open("test.json","r").read())

print(j['data'])

