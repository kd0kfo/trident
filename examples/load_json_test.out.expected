[{u'ref_start': 12, u'similarity': 63.16, u'query_start': 3, u'query_end': 21, u'score': 156.0, u'query_id': u'mir1', u'reference_id': u'mir1idealizedpurinestrand5to3', u'ref_end': 12, u'identity': 63.16, u'energy': -185.88, u'alignment_length': 19}, {u'ref_start': 13, u'similarity': 75.0, u'query_start': 2, u'query_end': 21, u'score': 240.0, u'query_id': u'mir1', u'reference_id': u'mir1idealizedpurinestrand5to3', u'ref_end': 13, u'identity': 75.0, u'energy': -249.51, u'alignment_length': 20}]
